src/createImports.mts

```diff
@@ -45,7 +45,23 @@ export const createImports = ({
           ts.factory.createImportSpecifier(
             true,
             undefined,
-            ts.factory.createIdentifier("QueryClient"),
+            ts.factory.createIdentifier("Options")
+          ),
+        ])
+      ),
+      ts.factory.createStringLiteral("@hey-api/client-fetch"),
```

Comment:
Could you consider importing `@hey-api/client-fetch` or `@hey-api/client-axios` depending on the `client` option?

Reply:
added the check with ternary. it should be OK now.